headers/types/common/common.h

```diff
@@ -1213,4 +1213,32 @@ ASSERT_SIZE(struct thread_info, 52); // Could be shorter
 // TODO: Add more data file structures, as convenient or needed, especially if the load address
 // or pointers to the load address are known.
 
+// State of the screen fade in all other modes except dungeon mode
+#pragma pack(push, 1)
```

Comment:
I don't think this is normally used for structs, but I don't know much about how packing works. Why is it there? Is this about the failed size checks you mentioned on Discord? Can't that be solved by explicitly declaring the padding fields?

Reply:
Yeah, basically [here](https://github.com/UsernameFodder/pmdsky-debug/blob/master/headers/README.md#struct-packing) there are given two options, either to pack it like this for no padding or declare padding manually. I guess in this case it doesn't matter since it's at the end of the struct so maybe manual padding would make more sense?

Reply:
It depends on how sure we are about the exact size of the struct.

In general, packed structs are uncommon, and in my experience are typically only used when dealing with raw file formats, low-level data processing, and other things like that. Sometimes we can be absolutely sure of a struct's size (e.g., we see it allocated via `MemAlloc`), but in this case, it seems like this struct is used for some globals and the size isn't super obvious to me, so I would probably opt for explicit padding at the end here.